dev/tasks/verify-rc/github.nix.yml

```diff
@@ -45,12 +45,16 @@ jobs:
 
           {{ flag }}
           if [ $(uname) = "Darwin" ]; then
-            brew install llvm@7 thrift glib
+            brew install llvm@8 thrift
+            brew reinstall glib
```

Comment:
Can we use the followings here?

```bash
brew bundle --file=arrow/cpp/Brewfile
brew bundle --file=arrow/c_glib/Brewfile
```

Reply:
Well, it failed later at least: https://github.com/ursa-labs/crossbow/runs/596869002?check_suite_focus=true

Reply:
It seems that gobject-introspection is old (1.64.0). 1.64.1 is the latest.
I've added `brew update`.